pages/courses/[id].js

```diff
@@ -157,7 +158,20 @@ function Course({ course, currentDate }) {
   const styleImageCover = {
     borderRadius:'10px'
   }
+  
+  const [kickoffStartDate,setKickoffStartDate] = useState()
+  const [kickoffEndDate,setKickoffEndDate] = useState()
 
+
+  useEffect(() => {
+    if (!cohort?.kickoffStartTime && !cohort?.kickoffEndTime) return
+    const cohortKickoffDateGMTPattern = dateFormat(cohort?.kickoffStartTime)
```

Comment:
Is it needed the `?` op when we pass it to `dateFormat` function? 
This is more of a question, but doesnt it check for it before on the `if` statement?